### PR TITLE
chore: remove chunked file read + debug log view

### PR DESCRIPTION
**Pull request by jeanregisser**
### Description

Follow up from https://github.com/valora-inc/wallet/pull/4633#pullrequestreview-1780234436, the (buggy) chunked file read can now be removed. Along with the debug log view which was using it.

### Test plan

- Tests pass

### Related issues

- Fixes RET-948

### Backwards compatibility

Yes

### CODE REVIEW

File: src/app/Debug.tsx [hunk 1]
```diff
@@ -33,9 +33,9 @@ export class Debug extends React.Component<RootState, State> {
   }
 
   updateLogs = async () => {
-    const reactNativeLogs = await Logger.getMonthLogs()
     this.setState({
-      reactNativeLogs: reactNativeLogs || 'Not Found',
+      reactNativeLogs:
+        "This feature has been removed. It didn't work with log files above a certain size.",
```
Comment (jeanregisser) [inline src/app/Debug.tsx:38]: I don't think anyone on the team is really using that debug screen to read logs. But let me know if you do.

Comment (cajubelt) [reply src/app/Debug.tsx:38]: can you just remove the feature? this seems like it leaves code hanging around that doesn't actually do anything anymore

**Review by cajubelt — CHANGES_REQUESTED** [review]

File: src/app/Debug.tsx (hunk 1 above)
Comment (jeanregisser) [reply src/app/Debug.tsx]: Yes, I first wanted to do this separately, but the whole log view is now removed.

**Review by kathaypacific — APPROVED** [review]
🧹

**Pull request merged**